### PR TITLE
disable social media links on url not provided closes #115

### LINKED ISSUE #115: Disable social links on particpant card if the link isn't available.

### PR DESCRIPTION
**Pull request by sourabh1031**
closes #115

### CODE REVIEW

**Comment by sourabh1031** [comment]
screenshot from 2019-01-18 14-42-26

**Comment by sourabh1031** [comment]
@prateek76 please review

File: js/main.js [hunk 1]
```diff
@@ -116,6 +116,75 @@ $(document).ready(function () {
...
+            if (
+                /^ *$/.test(participant.facebook) ||
+                participant.facebook == ""
+            ) {
+                //regex to check empty string or spaces
+                participant.facebook = "#";
+
+            } else {
+                var url = participant.facebook;
+                console.log(url);
```
Comment (pradeepgangwar) [inline js/main.js]: Remove this console.log()

File: js/main.js [hunk 2]
```diff
@@ -116,6 +116,75 @@ $(document).ready(function () {
...
+            if (
+                /^ *$/.test(participant.github) ||
+                participant.github == ""
+            ) {
+                //regex to check empty string or spaces
+                participant.github = "#";
+
+            } else {
+                var url = participant.github;
+                console.log(url);
```
Comment (pradeepgangwar) [inline js/main.js]: remove this too.

File: js/main.js [hunk 3]
```diff
@@ -116,6 +116,75 @@ $(document).ready(function () {
...
+            if (
+                /^ *$/.test(participant.twitter) ||
+                participant.twitter == ""
+            ) {
+                //regex to check empty string or spaces
+                participant.twitter = "#";
+
+            } else {
+                var url = participant.twitter;
+                console.log(url);
```
Comment (pradeepgangwar) [inline js/main.js]: same as above.

**Comment by pradeepgangwar** [comment]
> @prateek76 please review

Please provide a live link.

**Comment by sourabh1031** [comment]
> > @prateek76 please review
> 
> Please provide a live link.

you mean gif

**Comment by sourabh1031** [comment]
> > @prateek76 please review
> 
> Please provide a live link.

https://sourabh1031.github.io/Opencode-Collaborative-19/

**Comment by sourabh1031** [comment]
@pra

**Comment by sourabh1031** [comment]
@pra

**Comment by sourabh1031** [comment]
@prateek76 i think it is done now

**Pull request merged**